apps/test/unit/lib/levelbuilder/SaveBarTest.jsx

```diff
@@ -5,27 +5,29 @@ import sinon from 'sinon';
 import SaveBar from '@cdo/apps/lib/levelbuilder/SaveBar';
 
 describe('SaveBar', () => {
-  let defaultProps, handleSave;
+  let defaultProps, handleSave, handleView;
   beforeEach(() => {
     handleSave = sinon.spy();
+    handleView = sinon.spy();
     defaultProps = {
       isSaving: false,
       error: null,
       lastSaved: null,
-      handleSave
+      handleSave,
+      handleView
     };
   });
 
   it('renders default props', () => {
     const wrapper = mount(<SaveBar {...defaultProps} />);
-    expect(wrapper.find('button').length).to.equal(2);
+    expect(wrapper.find('button').length).to.equal(3);
     expect(wrapper.find('FontAwesome').length).to.equal(0); //spinner isn't showing
   });
 
   it('can save and keep editing', () => {
     const wrapper = mount(<SaveBar {...defaultProps} />);
 
-    const saveAndKeepEditingButton = wrapper.find('button').at(0);
+    const saveAndKeepEditingButton = wrapper.find('button').at(1);
```

Comment:
please try changing this and any other test cases to use `shallow` instead of `mount`. I see that `simulate` is available [here](https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/shallow.html), though I'll admit that I'm not 100% sure it will work in the way I'm expecting when shallow rendering.